guides/source/action_view_overview.md

```diff
@@ -14,18 +14,20 @@ After reading this guide, you will know:
 What is Action View?
 --------------------
 
-In Rails, web requests are handled by [Action Controller](action_controller_overview.html) and Action View. Typically, Action Controller is concerned with communicating with the database and performing CRUD actions where necessary. Action View is then responsible for compiling the response.
+Action View is the V in [MVC](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller). [Action Controller](action_controller_overview.html) and Action View work together to handle web requests. Action Controller is concerned with communicating with the model layer (of MVC) and retrieving data. Action View is then responsible for compiling a response to the web request using that data.
 
-Action View templates are written using embedded Ruby in tags mingled with HTML. To avoid cluttering the templates with boilerplate code, several helper classes provide common behavior for forms, dates, and strings. It's also easy to add new helpers to your application as it evolves.
+Action View templates (also referred to simply as "views") are written using Embedded Ruby (ERB), which allows using Ruby code within HTML documents.
```

Comment:
Good point @MatheusRich , for a new rubyst should be interesting to know how flexible the framework is.

Reply:
While this is true, I don't think it'ts good to encourage the use of Haml or Slim in the official guides. Maybe mention json or xml only?

Reply:
We mention this under "Templates": `Other libraries may add other template types and file extensions as well.`. It's okay to say that support can be extended by external libraries, but I don't think we need to be specific with Haml/Slim, because tomorrow another one might show up and we'd need to update, etc.

Reply:
I think the updated text is enough now. No need to cite explicit libraries here.